earthpy/io.py

```diff
@@ -267,7 +274,8 @@ def _download(self, url, path, kind, replace, verbose):
             return path
 
         if verbose is True:
```

Comment:
should we try: 
```
with LoggingContext(logger, logging.INFO if verbose else None):
    log.info("message")
```

Reply:
this is NOT printing is the issue right now. we want the logger to PRINT the output IF verbose is set to TRUE